pkg/networkservice/common/macaddr/server.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package macaddr provides chain element to set EthernetContext.SrcMacAddr
+package macaddr
```

Comment:
Why we cannot move this to sdk-kernel?

Reply:
It is not for kernel, it is for the ethernet context in general. So it can be used as for SR-IOV (kernel, VFIO) so for any other mechanism.

Reply:
I think it is can be used only sriov. Can you provide any use case when we need this for kernel case?